Dockerfile

```diff
@@ -1,6 +1,4 @@
-FROM ubuntu:xenial
-
-MAINTAINER  Andrej Antas <andrej@antas.cz>
+FROM ubuntu:bionic
 
```

Comment:
somebody has to be written somewhere, so people know who to address in communication about problems with image